beta-pipelines/skylab/ATAC/ATAC.wdl

```diff
@@ -50,6 +52,7 @@ workflow ATAC {
     min_map_quality: "the minimum mapping quality to be filtered by samtools view and snap-pre (snaptools task)"
     max_fragment_length: "the maximum fragment length for filtering out reads by gatk and snap-pre (snaptools task)"
     output_base_name: "base name to be used for the pipelines output and intermediate files"
+    bin_size_list: "space separated list of bins to generate"
```

Comment:
I'm not clear on what "space separated list" means. Isn't this just a numerical value regarding the number of bins? No reason to block this merge to master on this comment but maybe we can follow up on it. @ekiernan @sophiacrennan

Reply:
In the README we say: "List of bin sizes used to generate cell-by-bin matrices"- is that clear enough? We confirmed that this option can be a list of values, not just a single value.

Reply:
That's super interesting, I didn't realize it could be a list! I think the meta is ok as it is then, although I do think the README wording is a bit clearer.